### DISCUSSION TITLE
stop-loss orders

### DISCUSSION THREAD

**Discussion by elnaz-gharoon** [SE-TINF22B2/G3-ApeRepublic#19] [User-Stories]
Als vorsichtiger Trader qwq will ich Stop-Loss-Orders setzen können, um meine potenziellen Verluste zu begrenzen.

**Comment by Marc738** [comment]
Ich finde das ist etwas speziell. Man könnte eher sagen das man Orders einbaut die auf gewisse Ereignisse reagieren, wie das erreichen eines Kurses.
Dann würde diese User Story auch buy-limit order
